templates/birds/utility.macros.j2

```diff
@@ -5,12 +5,14 @@
     <div class="h-card">
       {{ utility.icon(page.config.bird, size='face') }}
 
-      {{ page.render_summary() }}
+      <div class="bio-short">
```

Comment:
I wonder if we want to use full bios now that they are at the bottom of the page?